src/sql-parser/src/parser.rs

```diff
@@ -3716,6 +3717,21 @@ impl<'a> Parser<'a> {
         }
     }
 
+    fn parse_kafka_include_header(&mut self) -> Result<SourceIncludeMetadata, ParserError> {
+        let key = self.parse_literal_string()?;
+        let alias = self
+            .parse_keyword(AS)
+            .then(|| self.parse_identifier())
+            .transpose()?;
+        let use_bytes = self.parse_keyword(BYTES);
```

Comment:
Slight preference for this to be `FORMAT BYTES` instead of just `BYTES`. Also we should accept `FORMAT TEXT` as an explicit way to set the other format to make it easier for users to auto-generate queries where they always specify a format

Reply:
Yes, I've been borrowing the [`WEBHOOK` syntax](https://materialize.com/docs/sql/create-source/webhook/#syntax) here. If we think we should fix the syntax now, I'm happy to make the change. But we should then also modify the syntax for Webhook, which is currently `INCLUDE HEADER <name> AS <ident> [BYTES]`.